client/state/jetpack-settings/modules/test/selectors.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+import {
+	isActivatingModule,
+	isModuleActive
+} from '../selectors';
+
+import {
+	modules as MODULES_FIXTURE,
+	requests as REQUESTS_FIXTURE
+} from './fixture';
+
+describe( 'requests selectors', () => {
```

Comment:
Some selectors might not be based on requests / items. So it would be best to avoid this grouping. Instead, let's add a single `describe( 'selectors', () => {` that wraps all selectors.